pkg/mtbroker/filter/filter_handler.go

```diff
@@ -184,6 +186,18 @@ func (r *Handler) serveHTTP(ctx context.Context, event cloudevents.Event, resp *
 		return errors.New("unable to parse path as a Trigger")
 	}
 
+	ctx, span := trace.StartSpan(ctx, tracing.TriggerMessagingDestination(triggerRef.NamespacedName))
+	defer span.End()
+
+	if span.IsRecordingEvents() {
+		span.AddAttributes(
+			tracing.MessagingSystemAttribute,
+			tracing.MessagingProtocolHTTP,
+			tracing.TriggerMessagingDestinationAttribute(triggerRef.NamespacedName),
+			tracing.MessagingMessageIDAttribute(event.ID()),
+		)
+	}
```

Comment:
move to some util?  It's like 4times in this PR :man_shrugging:

Reply:
I'm not sure a helper function would improve readability. All of these attributes besides the system attribute would need to be arguments to the helper function anyways and I prefer being explicit about what attributes are being added.